airbyte-integrations/connectors/source-bing-ads/source_bing_ads/source.py

```diff
@@ -597,6 +597,13 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             Campaigns(client, config),
         ]
 
+        # Added in the case of task #12489
+        # https://github.com/airbytehq/airbyte/issues/12489
+        config['hourly_reports'] = client.hourly_reports
+        config['daily_reports'] = client.daily_reports
+        config['weekly_reports'] = client.weekly_reports
+        config['monthly_reports'] = client.monthly_reports
+
         if config["hourly_reports"] or config["daily_reports"] or config["weekly_reports"] or config["monthly_reports"]:
```

Comment:
shouldn't we just include this stream by default instead of having this if statement?

Reply:
same question for the ones below